special-interest-groups/sig-dashboard/roles-and-organization-management.md

```diff
@@ -0,0 +1,75 @@
+# Dashboard SIG Roles and Organization Management
+
+## Join and Promotion
+
+We mainly follow the instructions in [TiDB Developer Group](../../architecture/README.md#tidb-developer-group).
+It lists common requirements, responsibilities and privileges of each role in developer group.
+
+### Join
+
+There is no restrictions to join the discussion of Dashboard SIG. See [README](./README.md) for where we discuss.
+
+However you will be only listed in our [member list] if you have **Active Contributor** role or above.
+
+### Promotion
+
+#### Concept: PR Point
```

Comment:
By introducing PR point, there is a lot of things we need to consider:

1. how do they know how many PR points they have?
2. does other SIG have PR point?
3. how to distinguish different PR points between different medium or hard PRs?

I think we'd better to keep the current promotion rule listed in [here](https://github.com/pingcap/community/tree/master/architecture#tidb-developer-group)

Reply:
Although PR point is not precise enough, the current promotion rule is worse.